src/wast-parser.cc

```diff
@@ -762,6 +762,13 @@ Result WastParser::ErrorIfLpar(const std::vector<std::string>& expected,
 
 bool WastParser::ParseBindVarOpt(std::string* name) {
   WABT_TRACE(ParseBindVarOpt);
+  if (PeekIsAnnotation("name")) {
```

Comment:
Is it okay that this is in `ParseBindVarOpt`? I'm not sure if the `@name` annotation is valid in every place that it's used.